src/locale/de_DE.js

```diff
@@ -42,10 +42,10 @@ export const de_DE = {
   ERR_PACKAGE_NO_RUNTIME: 'Paketlaufzeit \'{0}\' nicht gefunden',
   ERR_PACKAGE_NO_METADATA: 'Metadaten für \'{0}\' nicht gefunden. Ist es in der Manifest?',
   ERR_PACKAGE_EXCEPTION: 'Eine Ausnahme trat in \'{0}\' auf.',
-  ERR_WINDOW_ID_EXISTS: 'Window with ID \'{0}\' already exists',
+  ERR_WINDOW_ID_EXISTS: 'Fenster mit dem ID \'{0}\' existiert schon',
```

Comment:
'Fenster mit der ID \'{0}\' existiert schon'.
"der" instead of "dem"

Reply:
yes... didn't remmeber if it was die ID or der ID... :/